module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```diff
@@ -1998,6 +1999,7 @@ public function renewMyItems($renewDetails)
     {
         $details = [];
         foreach ($renewDetails['details'] as $detail) {
+            $username = $renewDetails['patron']['cat_username'];
```

Comment:
This value doesn't appear to change, so would it be better to set it above the foreach loop?

Reply:
You are right, it is fixed now